universal-application-tool-0.0.1/app/services/applicant/question/ApplicantQuestion.java

```diff
@@ -76,6 +76,20 @@ public boolean isOptional() {
     return programQuestionDefinition.optional();
   }
 
+  /**
+   * Return true this question is required but was left unanswered while filling out the current
+   * program.
+   */
+  public boolean isRequiredButWasUnansweredInCurrentProgram() {
+    return !isOptional() && !errorsPresenter().isAnswered() && wasRecentlyUpdated();
+  }
+
+  /** Returns true if this question was most recently updated in this program. */
+  private boolean wasRecentlyUpdated() {
```

Comment:
super nit (non-blocking): Could we call this `wasUpdatedInThisProgram`, since that's really what it's checking? We're not checking timestamps here so "recently" is a bit confusing

Reply:
fixed